django/contrib/admin/tests.py

```diff
@@ -123,13 +123,21 @@ def wait_page_loaded(self, timeout=10):
         """
         Block until a new page has loaded and is ready.
         """
+        from selenium.common.exceptions import WebDriverException
         from selenium.webdriver.common.by import By
         from selenium.webdriver.support import expected_conditions as ec
 
         old_page = self.selenium.find_element(By.TAG_NAME, "html")
         yield
         # Wait for the next page to be loaded
-        self.wait_until(ec.staleness_of(old_page), timeout=timeout)
+        try:
+            self.wait_until(ec.staleness_of(old_page), timeout=timeout)
+        except WebDriverException:
+            # Issue in recent versions of Chrome driver where a WebDriverException
+            # error is raised rather than a StaleElementReferenceException, see:
+            # https://issues.chromium.org/issues/42323468
+            pass
```

Comment:
Another way to handle this could be to install a specific chrome driver in the GitHub action
This relates to these errors: https://github.com/django/django/actions/runs/13832588256/job/38700091922?pr=19267